src/Rector/Dynamic/MethodNameReplacerRector.php

```diff
@@ -118,7 +134,7 @@ public function refactor(Node $node): ?Node
             return $this->resolveClassRename($node, $oldToNewMethods, $methodName);
         }
 
-        $node->name = new Identifier($oldToNewMethods[$methodName]);
+        $this->methodNameChanger->renameNode($node, $oldToNewMethods[$methodName]);
```

Comment:
@TomasVotruba The current errors are coming from here. Should we change some logic?